editor/editor_help.cpp

```diff
@@ -2618,20 +2618,25 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 			p_rt->set_cell_padding(Rect2(10 * EDSCALE, 10 * EDSCALE, 10 * EDSCALE, 10 * EDSCALE));
 			p_rt->push_color(code_dark_color);
 
-			if (lang.is_empty() || lang == "gdscript") {
+			bool highlighted = false;
+
 #ifdef MODULE_GDSCRIPT_ENABLED
+			if (!highlighted && (lang.is_empty() || lang == "gdscript")) {
 				EditorHelpHighlighter::get_singleton()->highlight(p_rt, EditorHelpHighlighter::LANGUAGE_GDSCRIPT, codeblock_text, is_native);
-#else
-				p_rt->add_text(codeblock_text);
+				highlighted = true;
+			}
 #endif
-			} else if (lang == "csharp") {
+
 #ifdef MODULE_MONO_ENABLED
+			if (!highlighted && lang == "csharp") {
 				EditorHelpHighlighter::get_singleton()->highlight(p_rt, EditorHelpHighlighter::LANGUAGE_CSHARP, codeblock_text, is_native);
-#else
-				p_rt->add_text(codeblock_text);
+				highlighted = true;
+			}
 #endif
-			} else {
+
+			if (!highlighted) {
 				p_rt->add_text(codeblock_text);
+				highlighted = true;
```

Comment:
To be pedantic, in this case the text is **not** highlighted. I would call the variable `printed` or something like that.